### PR TITLE
Fix dependency

### LINKED ISSUE #76: Wrong mypy dependency version
While installing dependencies you get:

```
ERROR: django-stubs 1.0.2 has requirement mypy<0.730,>=0.720, but you'll have mypy 0.700 which is incompatible. 
```

### PR DESCRIPTION
**Pull request by grzegorz225**
Closes #76

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,5 +10,5 @@ dj-static
 raven
 python-dateutil
 Faker
-mypy==0.700
+mypy
```
Comment (cytadela8) [inline requirements.txt]: What if mypy gets updated, but django-stubs not?

File: requirements.txt [hunk 2]
```diff
@@ -1,14 +1,14 @@
-Django<2.0
+Django==1.11.22
```
Comment (cytadela8) [inline requirements.txt]: > What about **no**?

We shouldn't add such precise dependencies in `requirements.txt`. Minor version updates should be allowed. Could you tell me why the issue I showed happens?

File: type_check.sh [hunk 3]
```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
-mypy -p wwwapp
+DJANGO_SETTINGS_MODULE=wwwapp.settings mypy -p wwwapp
```
Comment (krzys-h) [inline type_check.sh]: Why is this required both here and in mypy.ini?

Comment (grzegorz225) [reply type_check.sh]: It's an open issue with django-stubs, without this environment variable analysis fails because some django code is unable to find some parts of the configuration.
